travelpal/lib/travelpal_web/router.ex

```diff
@@ -13,6 +14,13 @@ defmodule TravelpalWeb.Router do
     plug :accepts, ["json"]
   end
 
+  def get_current_user(conn, _params) do
```

Comment:
Need to remove this since we're using tokens, and `Accounts` doesn't exist anymore after switching to JSON resources

Reply:
Oops idk why that's still there